front-end/src/pages/HomePage.tsx

```diff
@@ -16,7 +23,44 @@ export default function HomePage(props:RouteComponentProps) {
 
   function searchUsers(e:any){
     e.preventDefault();
-    props.history.push('/authors/'+userSearch);
+    //props.history.push('/authors/'+userSearch);
+    axios.get('http://localhost:8000/api/author/' + userSearch + "/")
```

Comment:
Use `process.env.REACT_APP_API_URL + "/api/author/"` instead of `'http://localhost:8000/api/author/'`